frontend/src/component/common/ConstraintsList/ValuesList/ValuesList.tsx

```diff
@@ -0,0 +1,66 @@
+import type { FC } from 'react';
+import { styled, Tooltip } from '@mui/material';
+import {
+    Truncator,
+    type TruncatorProps,
+} from 'component/common/Truncator/Truncator';
+
+export type ValuesListProps = {
+    values?: string[];
+    tooltips?: Record<string, string | undefined>;
+} & Pick<TruncatorProps, 'onSetTruncated'>;
+
+const StyledValuesContainer = styled('div')({
+    flex: '1 1 0',
+});
+
+const StyledValueItem = styled('span')(({ theme }) => ({
+    padding: theme.spacing(0.25),
+    display: 'inline-block',
+    span: {
```

Comment:
What's the nested span do? Does the top-level one use padding for spacing other elements? Can't that be handled by the container and `gap`?

Reply:
the entire thing has to be `inline` for truncator to work

Reply:
Huh. So `inline-flex` doesn't work?